supabase/migrations/20240823182342_feedback_table.sql

```diff
@@ -0,0 +1,33 @@
+-- We use this to store feedback from our users
+CREATE TABLE IF NOT EXISTS public.feedback (
+    id SERIAL PRIMARY KEY,
+    email VARCHAR(255),
+    feedback_text TEXT NOT NULL,
+    route TEXT NOT NULL,
+    created_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP
+);
+
+-- Grant permissions on public.feedback
+ALTER TABLE IF EXISTS public.feedback OWNER TO postgres;
+GRANT ALL ON TABLE public.feedback TO anon;
+GRANT ALL ON TABLE public.feedback TO authenticated;
+GRANT ALL ON TABLE public.feedback TO service_role;
+GRANT ALL ON TABLE public.feedback TO postgres;
+
+GRANT SELECT ON TABLE public.feedback TO toiletmap_web;
+GRANT INSERT ON TABLE public.feedback TO toiletmap_web;
+GRANT USAGE, SELECT ON SEQUENCE public.feedback_id_seq TO toiletmap_web;
+
```

Comment:
not a review but a personal Q as I'm not so familiar with `GRANT`, what does limiting `toiletmap_web` to these exclude it from doing? :)

Reply:
By default the role will not be given rights to SELECT/INSERT/UPDATE/DELETE/EXECUTE/etc... on any given table. We use the `GRANT` system to selectively allowlist our `toiletmap_web` role to perform only the operations it needs in order to function as intended.

In this case we'd like to allow grant it:
- Select & insert permissions
- Select & usage on the feedback id sequence

Reply:
Examples of what it's excluded from doing..

- No UPDATE or DELETE permissions.
- It can't modify the table schema because it lacks ALTER permissions.
- It can't run functions or procedures on that table.
- It can't drop things